server/helpers/db.js

```diff
@@ -197,8 +201,45 @@ const getUsers = async () => {
     .orderBy("user_id", "asc");
 };
 
+const getUsersByRole = () => {
```

Comment:
a better name for this would `getUsersWithRoles` not byRole. `byRole` implies that you're filtering by role which you are not (you're just all the users info + their roles)